client/ayon_core/pipeline/editorial.py

```diff
@@ -179,6 +179,24 @@ def get_media_range_with_retimes(otio_clip, handle_start, handle_end):
     media_in = available_range.start_time.value
     media_out = available_range.end_time_inclusive().value
 
+    # Ensure image sequence media_ref source and
+    # available range are absolute.
+    media_ref = otio_clip.media_reference
+    if (
+        hasattr(otio.schema, "ImageSequenceReference")
+        and isinstance(media_ref, otio.schema.ImageSequenceReference)
+        and media_in != media_ref.start_frame
+    ):
+        media_in = media_ref.start_frame
+        media_out += media_ref.start_frame
```

Comment:
If we'll keep it, should this first reduce by `media_in` (I'm fully out of context BTW, never worked with OTIO).
```suggestion
        media_out = (media_out - media_in) + media_ref.start_frame + 1
```